cookie-banner-rules-list.json

```diff
@@ -107,7 +107,7 @@
     {
       "click": {
         "optIn": "button#onetrust-accept-btn-handler",
-        "optOut": ".ot-pc-refuse-all-handler, #onetrust-reject-all-handler",
+        "optOut": ".ot-pc-refuse-all-handler, #onetrust-reject-all-handler, .onetrust-close-btn-handler",
```

Comment:
I would just like to mention that I'm not a Mozilla employee, just a random guy on the Internet, and you do not have to listen to me, but I'm not really a fan of adding a selector of this kind to an existing rule that is already used to detect and handle cookie banners on many different websites.

Why? Because I have already encountered situations where some cookie consent managers treated this type of a user action as something like "eh, I don't care, take it out of my sight" and interpreted it as a permission to store cookies on the user's device. I'm not entirely sure that such behavior is compliant with the GDPR/CCPA/PIPEDA/whatever regulations, but that's how it is.

For this reason, to be safe, I would rather suggest creating a separate rule for fnac.be, fnac.ch, fnac.com, and fnac.pt, like this:
```json
{
  "id": "87e6c488-abd4-4adc-bbb5-f638e55471b5",
  "domains": ["fnac.be", "fnac.ch", "fnac.com", "fnac.pt"],
  "click": {
    "presence": "#onetrust-consent-sdk",
    "optOut": ".onetrust-close-btn-handler",
    "optIn": "#onetrust-accept-btn-handler"
  }
}
```

And fnac.es could end up in this semi-global OneTust rule if you are already at it.

Reply:
You misunderstood me. The problems I experienced some time ago when creating rules for this project were that some cookie consent managers treated closing a banner with an X button as permission to store cookies on the user's device. The CMP used was not OneTrust, so in this case, adding `.onetrust-close-btn-handler` should not cause such problems, but who knows? It will be safer to create a separate rule, I think.

The banner on fnac.es is already supported by Firefox, as support for this website is provided by the global OneTrust rule. So you're right, you don't need to add it anywhere.

Reply:
You misunderstood me. The problems I experienced some time ago when creating rules for this project were that some cookie consent managers treated closing a banner with an X button as permission to store cookies on the user's device. The CMP used was not OneTrust, so in this case, adding `.onetrust-close-btn-handler` should not cause such problems, but who knows? It will be safer to create a separate rule, I think.

The banner on fnac.es is already supported by Firefox, as support for this website is provided by the global OneTrust rule. So you're right, you don't need to add it.